src/charm.py

```diff
@@ -185,7 +185,7 @@ def instance_map(self) -> typing.Optional[typing.Dict]:
         logger.debug("instance_map is: %s", str(instance_map))
         return instance_map
 
-    def change_config(self, charm_state: CharmState) -> None:
+    def reconcile(self, charm_state: CharmState) -> None:
         """Change configuration.
```

Comment:
I would expand the description here that this is the main entry for changes that require a restart, good to have it documented and serve as guidance (and reminder) for future additions/changes to the code base.

Reply:
sure,done.